orbital-camera.ts

```diff
@@ -60,8 +64,11 @@ export class OrbitalCamera extends Component {
     private _azimuthSpeed: number = 0;
     private _polarSpeed: number = 0;
 
-    start(): void {
+    init() {
         this.object.getPositionWorld(this._origin);
```

Comment:
`start()` or maybe even `onActivate()` (to re-set the origin on reactivation) would probably make the most sense for this

Reply:
It's a very deliberate choice to put it in `init`. If the component starts deactivated and the camera is moved the origin will be set to an incorrect value when the component is activated and `start` ran for the first time, if the origin is set at another moment that in `init`. This is exactly the scenario where I needed this component.

Reply:
Using object and components in `init()` is forbidden though. It works but could break anytime. I guess we need to think about activation order @Squareys

Reply:
Any suggestions how this should work without having it in `init()`? Another component disables this one from its `start()` and `start()` on this component is never called until it is enabled.